frontend/src/components/settings/ActivateCourseDialog/ActivateCourseDialog.tsx

```diff
@@ -26,7 +27,8 @@ const ActivateCourseDialog: FC<ActivateCourseDialogProps> = ({ course, open, onC
         code: course.code,
         title: course.title,
         term: course.term,
-        autoApproveRequests: course.autoApproveRequests,
+        autoApproveRequests: course.config.sharePeopleListWithStudents,
```

Comment:
copy and paste mistake? should be set to `course.config.autoApproveRequests` right